rest-api-with-fastify/app.js

```diff
@@ -5,9 +5,11 @@ import { Romcal } from 'romcal';
 import * as GeneralRoman from '@romcal/calendar.general-roman';
 import { France_Fr as FranceFR } from '@romcal/calendar.france';
 
+const LOG_LEVEL = process.env.ROMCAL_LOG_LEVEL || 'fatal';
```

Comment:
I think it might be nice to describe `ROMCAL_LOG_LEVEL` in `README.md`, as you do it for `ROMCAL_APP_PORT` in these files:

- https://github.com/romcal/romcal-examples/pull/6/files/746b2ff246f710523a96b6d8021be535b9441d77..6292bab2204fe5405d03a2cc46d9372370a5fa68#diff-af5701807f29a58d5ce48123bb3f9758232ddd0e75d04de900e5d3d78bef3495;
- https://github.com/romcal/romcal-examples/pull/6/files/746b2ff246f710523a96b6d8021be535b9441d77..6292bab2204fe5405d03a2cc46d9372370a5fa68#diff-74aaf4e5b3e2c607a1f1531a284eb65dfda46091580714b830ccf235f20f1ecc;
- https://github.com/romcal/romcal-examples/pull/6/files/746b2ff246f710523a96b6d8021be535b9441d77..6292bab2204fe5405d03a2cc46d9372370a5fa68#diff-33de137f63c8ec7f5c8be8395bca53156460cd1fc280cd132fab97c4944781f0.

Reply:
added